client/app/components/calendar/TimeIndicatorRow.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import moment from 'moment';
+import instanceOfMoment from 'proptypes/moment';
+import style from './calendar.scss';
+
+export default class TimeIndicatorRow extends React.Component {
+  static propTypes = {
+    days:                     React.PropTypes.arrayOf(instanceOfMoment).isRequired,
+    unitEventLengthInSeconds: React.PropTypes.number.isRequired,
+    rowHeight:                React.PropTypes.string
+  };
+  static defaultProps = {
+    rowHeight: style.rowHeight
+  };
+
+  componentDidMount() {
+    this.intervalId = setInterval(() => {
+      this.forceUpdate();
+    }, 60 * 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.intervalId);
+  }
+
+  render() {
+    return (
+      <tr id="time-indicator-row"><td className="time-cell" />{this._timeIndicatorCells()}</tr>
+    );
+  }
+
+  _isToday(day) {
+    return moment(day).isSame(moment(), 'day');
+  }
+
+  _topVal() {
+    const minutesFromMidnight = moment().diff(moment().clone().startOf('day'), 'minutes');
```

Comment:
No need to call `clone()` on `moment()` - `moment()` already creates a new instance of Moment